### ISSUE TITLE
Packet::Disconnect is not implemented

### ISSUE THREAD

**Issue by bbigras** [AtherEnergy/rumqtt#130]
rumqtt 67d1bb465b34bf8ccf2a058fbcb4cbb5ff919ceb

```
thread '<unnamed>' panicked at 'not yet implemented', /home/bbigras2/.cargo/git/checkouts/rumqtt-97ac1717aa98c207/67d1bb4/src/client/mqttstate.rs:71:18
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

The packet is `Packet::Disconnect`. Because I call `client.disconnect()`.

**Comment by denizs** [comment]
Implemented in #135

**Comment by tekjar** [comment]
Thanks @denizs. Appreciate your help. Keep contributing
